python/ml4ir/applications/ranking/model/losses/listwise_losses.py

```diff
@@ -32,7 +35,104 @@ def _loss_fn(y_true, y_pred):
             y_pred : [batch_size, num_classes]
             mask : [batch_size, num_classes]
             """
-            return cce(y_true, tf.math.multiply(y_pred, mask))
+
+            #Fixme
+            """
+            Queries with ties in the highest scores would have multiple one's in the 1-hot vector.
```

Comment:
Should we just scale it by the number of docs with highest score?

Reply:
Why do you think that would be useful?

Reply:
@mohazahran , ping on this

Reply:
I don't quite get your point. Scaling by the number of ties, that would make the loss bigger for cases with many ties, right? How's that would help.

Reply:
Ah, I see the ambiguity. I meant to scale down (1/num_ties )* loss